src/modules/commander/commander.cpp

```diff
@@ -2502,10 +2510,18 @@ set_main_state_rc(struct vehicle_status_s *status_local, struct manual_control_s
 				res = main_state_transition(status_local,vehicle_status_s::MAIN_STATE_STAB);
 			}
 
-		} else {
+		} 
+		if(sp_man->rattitude_switch == manual_control_setpoint_s::SWITCH_POS_ON){
```

Comment:
This seems wrong. The rattitude switch value will default to non implemented, which means anyone not having configured the switch gets manual here. Looking at it using an else if would fix this, as rattitude is only selected if acro is not configured.